extension/content_script.js

```diff
@@ -490,6 +489,7 @@ In the INACTIVE state, the widgets are not visible, and no event handlers are at
 function oneTimeSetup() {
 	let readableDomEls = window.parseDocument();
 	doc = new Doc(readableDomEls);
+	doc.updateWordModel(readableDomEls);
```

Comment:
Why not place updateWordModel() in constructor so that we won't ever forget to call it when initializing?

Reply:
👍